src/main/java/uk/gov/hmcts/reform/civil/model/hearing/HFPbaDetails.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.civil.model.hearing;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+import uk.gov.hmcts.reform.civil.model.Fee;
+import uk.gov.hmcts.reform.civil.model.PaymentDetails;
+import uk.gov.hmcts.reform.civil.model.common.DynamicList;
+
+import java.time.LocalDateTime;
+
+@Setter
+@Data
+@Builder(toBuilder = true)
+public class HFPbaDetails {
+
+    private final DynamicList applicantsPbaAccounts;
+    private final String pbaReference;
+    private final Fee fee;
+    private final PaymentDetails paymentDetails;
+    private final LocalDateTime paymentSuccessfulDate;
+    private final String serviceReqReference;
+
+    @JsonCreator
+    HFPbaDetails(@JsonProperty("applicantsPbaAccounts") DynamicList applicantsPbaAccounts,
+                 @JsonProperty("pbaReference") String pbaReference,
+                 @JsonProperty("fee") Fee fee,
+                 @JsonProperty("paymentDetails") PaymentDetails paymentDetails,
+                 @JsonProperty("paymentSuccessfulDate") LocalDateTime paymentSuccessfulDate,
+                 @JsonProperty("serviceRequestReference") String serviceReqReference) {
+        this.applicantsPbaAccounts = applicantsPbaAccounts;
+        this.pbaReference = pbaReference;
+        this.fee = fee;
+        this.paymentDetails = paymentDetails;
+        this.paymentSuccessfulDate = paymentSuccessfulDate;
+        this.serviceReqReference = serviceReqReference;
+
+    }
+}
```

Comment:
I believe this can be simplified.
Take a look at the classes annotated with `@Jacksonized`.

Is there a particular reason why you chose this approach instead of just putting the jackson annotations on the individual properties and do away with the dedicated constructor?